docs/content/preview/launch-and-manage/monitor-and-alert/metrics/cache-storage.md

```diff
@@ -12,6 +12,8 @@ menu:
 type: docs
 ---
 
+# RocksDB storage subsystem metrics
```

Comment:
Heading 2 (##) is the highest docs uses in pages - you need to put this to ## and bump its children down a level (###)

Reply:
Fixed this